example/package.json

```diff
@@ -37,30 +37,29 @@
     ]
   },
   "dependencies": {
-    "@cumulus/api": "1.16.0",
-    "@cumulus/checksum": "1.16.0",
-    "@cumulus/cmrjs": "1.16.0",
-    "@cumulus/common": "1.16.0",
-    "@cumulus/deployment": "1.16.0",
-    "@cumulus/discover-granules": "1.16.0",
-    "@cumulus/discover-pdrs": "1.16.0",
-    "@cumulus/files-to-granules": "1.16.0",
-    "@cumulus/hello-world": "1.16.0",
-    "@cumulus/integration-tests": "1.16.0",
-    "@cumulus/move-granules": "1.16.0",
-    "@cumulus/parse-pdr": "1.16.0",
-    "@cumulus/pdr-status-check": "1.16.0",
-    "@cumulus/post-to-cmr": "1.16.0",
-    "@cumulus/queue-granules": "1.16.0",
-    "@cumulus/queue-pdrs": "1.16.0",
-    "@cumulus/sf-sns-report": "1.16.0",
-    "@cumulus/sync-granule": "1.16.0",
-    "@cumulus/test-processing": "1.16.0",
-    "aws-sdk": "^2.585.0",
-    "ssh2-streams": "^0.4.8"
```

Comment:
Seems like test are passing so I'm guessing that this package removal is intentional?

Reply:
🤔